serial_driver/include/serial_driver/serial_driver_node.hpp

```diff
@@ -59,12 +59,33 @@ enum class stop_bits_t
   two
 };
 
+template<typename Derived>
```

Comment:
For now it's ok to have this helper template here, but it'd be best to move this to a separate package, or release `autoware_auto_common` separately from the Autoware.Auto repo. Could you add a `TODO` comment here so we'll address this later? Thanks.

Reply:
Added.